docs/contributing.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+We're thrilled that you're considering contributing to next-on-pages, all contributions are greatly appreciated, regardless if they are simple issue reporting, bug fixes, documentation changes, etc...
+
+No contribution is too small and all help moving the project forward!
+
+## Submitting an issue or feature request
+
+If you find an bug or want to request a feature open a [new issue](https://github.com/cloudflare/next-on-pages/issues/new) documenting it, but first please check that:
+
+- there isn't already an issue describing the same bug of feature request, if there is please contribute to the conversation there (as this helps keeping all the information in a single place and also shows which bugs/features people are most interested in).
+- if you've found a Next.js missing feature, make sure that it is not already documented in the [Supported Versions and Features document](./supported.md).
+
+If you end up opening the issue make it as descriptive as possible, the more details you can provide the easier it will be for us to understand and handle it.
+
+In case of bugs very useful things to add are:
+
+- a minimal reproduction for it (both the steps to perform and the source code if possible)
+- what version of next-on-pages ad Next.js you're using
+- your operative system and Node version
+- what behavior you expected vs what behavior you got
+
+<!-- TODO: add a template for issues and simplify the above -->
+
+## Pull Requests
+
+When opening a new PR make sure to set up an informative title for it and provide as much details as you can in the PR's description so that it is clear what its intentions are. This helps in simplifying and speeding up the reviewing process.
+
+If your PR is addressing an existing issue make sure that it references the issue (as per the [Github PR Issues linking documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)) so that the two can be linked correctly (and the issue gets automatically closed on merge).
+
+Smaller PRs are preferred since they are easier to review and merge quickly, if you're planning to make multiple unrelated (or loosely related) changes please consider splitting them in multiple PRs.
```

Comment:
```suggestion
Smaller PRs are preferred since they are easier to review and merge quickly. If you're planning to make multiple unrelated (or loosely related) changes please consider splitting them in multiple PRs.
```